### ISSUE TITLE
App is too complex/has deviated from the goal

### ISSUE THREAD

**Issue by jeffreyssmith2nd** [EOSIO/eosio-project-demux-example#32]
Due to a lot of issues I noticed in mentoring the Sydney and London hackathons, I think this repo is too complex for the goals it is trying to achieve.

The point of this repository should be to introduce developers to demux and how to properly run demux against an EOSIO chain as well as properly using eosjs. By having opinionated frontend technologies (React and WebSockets) I think we are taking away from the overall goal.

I worked with multiple developers that wanted to use demux but were spending too much time on issues with the frontend code because it was using technologies they weren't familiar with, but also weren't comfortable/aware of the best approach to rip it out.

I think we should provide a simple HTML5/CSS/plain JS frontend with properly modularized eosjs code so that developers can use the framework/language of their choosing while still getting the initial setup/tutorial that we're trying to provide.

**Comment by TaraTritt** [comment]
Definitely up for simplifying this example. I have also seen other hackathon participants struggle with understanding this example

**Comment by yashwanth2804** [comment]
yeah as a newbie it's a steep learning curve,
better if demux concentrated or modular simple code could be better.
a flow chart or representional diagram how project work even understands the project as whole.

**Comment by pavja2** [comment]
I also agree. Trying to convert a previous project to Demux based on this and it's very hard to know what is essential to using Demux and what is essential to the specific blog application demo. Given that demux itself is only lightly documented, a clear bare-bones example would go a very long way.

**Comment by bl4ck4ndbr0wn** [comment]
i actually don't agree.Its simple as it is.
I think it should be designed to be a production version to allow devs to know the standards when developing client projects.

**Comment by cyberluke** [comment]
I also don't agree. What is the purpose of hackathon? Hackathon is not how developers should work. Hackathon is for computer monkeys, who just sweat in 24h on Redbull and do not have time to learn anything new. Better have production ready example.
